[#72755] 17.2 Backlogs changes  https://community.openproject.org/wor… (#22263)
opf/openproject: docs/user-guide/backlogs-scrum/work-with-backlogs/README.md

## Create a new user story

In order to create a new work package in the *Backlogs* module, click on the **More (three dots)** icon in the top right corner or a Sprint or Bug Backlog and choose *New Story* from the drop-down menu.

![Create a new user story from a product backlog in Backlogs module in OpenProject](openproject_user_guide_backlogs_new_user_story.png)

> A pop-up form will appear to create a new work package. Here, you directly specify the work package type, subject and description. The list of types contains those work package types that are activated in the [System Administration](../../../system-admin-guide/backlogs/). Click **Create** to proceed.

Reviewer: ```suggestion
> A form dialog will appear to create a new work package. Here, you directly specify the work package type, subject and description. The list of types contains those work package types that are activated in the [System Administration](../../../system-admin-guide/backlogs/). Click **Create** to proceed.
```

Do we call these pop-ups elsewhere?

![A form to create a new work package in Backlogs module](openproject_user_guide_backlogs_new_wp_form.png)

> A new line will be added in the backlogs to show the newly created story.

Reviewer: ```suggestion
A new item will be added to the backlog to display the newly created story.
```

![newly created user story added to a sprint in Backlogs module in OpenProject](openproject_user_guide_backlogs_new_user_story_added.png)

> Clicking on a line within the sprint opens the work package detail view on the right side, so you can make adjustments while staying in the Backlogs module.

Reviewer: ```suggestion
> Clicking on an item within the sprint opens the work package detail view on the right side, so you can make adjustments while staying in the Backlogs module.
```
